modules/user_accounts/php/NDB_Form_user_accounts.class.inc

```diff
@@ -922,6 +922,9 @@ class NDB_Form_User_Accounts extends NDB_Form
      */
     private function _getEmailError($DB, $email)
     {
+        // remove illegal characters
+        $email = filter_var($email, FILTER_SANITIZE_EMAIL);
```

Comment:
FILTER_SANITIZE_EMAIL removes illegal characters but does not fix encoding issues as described in the redmine ticket

Reply:
@driusan I hate to say I was right but... I was right.

This addition breaks email addresses that contain HTML special chars such as `test@gmail.com>`

Reply:
It looks like at some point there might have been a bad merge, because here there is no if statement but in #8137 there is (resulting in filter_var being called twice)